sdk/state/payment.go

```diff
@@ -43,33 +44,28 @@ type CloseSignatures struct {
 	Declaration xdr.Signature
 }
 
+func (cas CloseSignatures) Set() bool {
```

Comment:
💡 `Set()` is slightly confusing to me. It seems like an action of setting something rather than getting information. What about `HasAllSignatures` like used in this [PR](https://github.com/stellar/experimental-payment-channels/pull/335)?

Reply:
also I know not part of this PR but could we change the[ line here](https://github.com/stellar/experimental-payment-channels/blob/afeeaf6b8b2aab887c1a05a52d389a0b6cab1c79/sdk/state/open.go#L41) to use `!= 0` as well for consistency?

Reply:
I agree, Set, is confusing. In this case we don't want all sigs, just any value. I'll change it Empty, like the other place we use Empty. Longer term maybe we should use pointers for these types. I regret going the other way now 😄 

Yup we can change the 0 condition check. I'll do that separately.

Reply:
c824d59

Reply:
I also fixed the 0 condition check too in 3016ad6.

Reply:
Addressing the 0 thing in https://github.com/stellar/experimental-payment-channels/pull/337.